File: src/main/java/ru/romanow/jpa/mapper/MapperConfiguration.java
package ru.romanow.jpa.mapper;

import org.mapstruct.MapperConfig;

import static org.mapstruct.InjectionStrategy.CONSTRUCTOR;
import static org.mapstruct.ReportingPolicy.ERROR;

@MapperConfig(componentModel = "spring", injectionStrategy = CONSTRUCTOR, unmappedTargetPolicy = ERROR)
public interface MapperConfiguration {
}

File: src/main/java/ru/romanow/jpa/mapper/config/MapperConfiguration.kt
package ru.romanow.jpa.mapper.config

import org.mapstruct.InjectionStrategy
import org.mapstruct.MapperConfig
import org.mapstruct.ReportingPolicy

@MapperConfig(
    componentModel = "spring",
    injectionStrategy = InjectionStrategy.CONSTRUCTOR,
    unmappedTargetPolicy = ReportingPolicy.ERROR
)
interface MapperConfiguration
